### PR TITLE
Fix: Travis build for Node

### PR DESCRIPTION
**Pull request by NishealJ**
## Description
Hi,

Earlier the travis nodejs build & tests for react were not running, they are running after these changes.
Also, few of the react tests are failing I'm debugging it  @sharmamohit123 can you please take a look :)?

The python build & tests are still now running, It seems an issue with pycurl I'm looking at this too.

Thanks!

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other

### Checklist:

- [x] My code follows the code style (modified PEP8) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by NishealJ** [comment]
an update, the react tests passed works fine!

**Comment by sharmamohit123** [comment]
thnx @NishealJ, I've looked at the changes, lgtm (problem seems to be incompatible node version with evergreen-ui). Also, I don't think that there is any version problem with pycurl. Issue is the missing `libgnutls28-dev ` library. Can you try adding it in the travis file and then doing a travis build?

**Comment by NishealJ** [comment]
> thnx @NishealJ, I've looked at the changes, lgtm (problem being the incompatible node version with evergreen-ui). Also, I don't think that there is any version problem with pycurl. Issue is the missing `libgnutls28-dev ` library. Can you try adding it in the travis file and then doing a travis build?

Yes, the libgnutls28-dev helps in running the backend tests :)

**Comment by NishealJ** [comment]
Hi @sharmamohit123,

Did a few changes, the test works fine, please take a look!

File: requirements/base.txt [hunk 1]
```diff
@@ -11,7 +11,8 @@ pexpect==4.6.0
 psutil==5.3.1
 psycopg2-binary==2.7.4
 PTP==0.4.2
-pycurl==7.43.0.2
+pycurl==7.43.0
+pyhamcrest
```
Comment (sharmamohit123) [inline requirements/base.txt]: @NishealJ are these changes required?

File: requirements/test.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
 mock>=1.3.0
 nose==1.3.7
-PyHamcrest==1.9.0
+PyHamcrest
```
Comment (sharmamohit123) [inline requirements/test.txt]: same here.

Comment (NishealJ) [reply requirements/test.txt]: Hi,
changed this to version 2.1.0, in my next commit!

File: requirements/base.txt (hunk 1 above)
Comment (NishealJ) [reply requirements/base.txt]: Hi @sharmamohit123 ,
while running the test at the local instance, we have to install the test.txt requirements and then run the tests(while running it locally it was suggesting to install pyhamcrest) we can update the dev docs.

I've removed this in my latest commits

**Comment by viyatb** [comment]
how is this coming along?

File: owtf/models/command.py [hunk 3]
```diff
@@ -17,7 +17,7 @@ class Command(Model):
     end_time = Column(DateTime)
     success = Column(Boolean, default=False)
     target_id = Column(Integer, ForeignKey("targets.id"))
-    plugin_key = Column(String, ForeignKey("plugins.key"))
+    plugin_key = Column(String)
```
Comment (sharmamohit123) [inline owtf/models/command.py]: @viyatb is there any other way we could solve the `Foreign key associated with column 'command_register.plugin_key' could not find table 'plugins'` error. This seems to be a good temporary fix but might cause problem elsewhere in the code. I tried a bunch of things but nothing worked out.

Comment (sharmamohit123) [reply owtf/models/command.py]: Rest of the changes seems fine.

**Comment by viyatb** [comment]
can you rebase?

File: owtf/models/command.py (hunk 3 above)
Comment (NishealJ) [reply owtf/models/command.py]: This is solved at #1040 Thanks!

**Comment by NishealJ** [comment]
> can you rebase?

Done, Please take a look!

**Comment by sharmamohit123** [comment]
@viyatb ready to be merged, thnx @NishealJ
